.buildkite/scripts/steps/lint.sh

```diff
@@ -9,7 +9,19 @@ source .buildkite/scripts/common/util.sh
 echo '--- Lint: stylelint'
 checks-reporter-with-killswitch "Lint: stylelint" \
   node scripts/stylelint
+echo "stylelint ✅"
```

Comment:
Added this so that BK shows how long this command actually takes, without it the logs just show `0s`